gpt4all-chat/src/chat.cpp

```diff
@@ -240,7 +240,7 @@ void Chat::newPromptResponsePair(const QString &prompt)
     m_chatModel->updateCurrentResponse(m_chatModel->count() - 1, false);
     // the prompt is passed as the prompt item's value and the response item's prompt
     m_chatModel->appendPrompt("Prompt: ", prompt);
-    m_chatModel->appendResponse("Response: ", prompt);
+    m_chatModel->appendResponse("Response: ", QString());
```

Comment:
The second parameter is no longer useful and can be removed. (The first argument is also always the same string as well - `m_chatModel->appendPrompt(prompt)` and `m_chatModel->appendResponse()` would convey the same information.)